Memory_management_algorithm/best_fit.cpp

```diff
@@ -0,0 +1,102 @@
+#include <bits/stdc++.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+using namespace std;
+
+void best_fit(int memory_block[], int process_size[], int m, int n) {
+     int allocated_list[n][3];
+     int allocation_marking[m];
+
+     for(int i = 0; i < m; i++)
+         allocation_marking[i] = -1;
+
+	for(int i = 0; i < n; i++) {
+	
+	    int allocation = -1;
+	    int mem_block = 10000000;
+
+	    for(int j = 0; j < m; j++) {
+	        if((process_size[i] <= memory_block[j]) && (allocation_marking[j] ==  -1)) {
+		    if(memory_block[j] < mem_block) {
+		        mem_block = memory_block[j];
+			allocation = j;
+		    }
+		}
+	    }
+		
+
+       	    if(allocation == -1) {
```

Comment:
inconsistency of indentation

Reply:
Inconsistency in indentation
use 4 space indentation

Reply:
The indentation is correct when I am viewing the code in my local editor while viewing it in Github, the indentation is changed. I don't know why it is happening! Does github changes the indentation?

@somya-kapoor

Reply:
![Screenshot_2020-03-19 Maxz2136 Algo_Ds_Notes](https://user-images.githubusercontent.com/41407039/77096878-f84f9f00-6a35-11ea-93bb-e04c0a531055.png)

Reply:
The above screenshot is when I see the code in the editor present in Github itself, the indentation seems correct but when viewed outside of editor, the indentation gets disturbed.

Reply:
you are using the tab indentation,
use space indentation